### PR TITLE
LPD-42060 AutoSF

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

File: modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/renderer/LanguageSelectFragmentRenderer.java [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+	}
+
+	@Override
+	public String getIcon() {
+		return "automatic-translate";
+	}
+
+	@Override
+	public String getLabel(Locale locale) {
+		return _language.get(locale, "language-select");
```
Comment (ealonso) [inline modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/renderer/LanguageSelectFragmentRenderer.java:87]: @victorg1991 As I can see the name of fragment in figma is "Localization Select",

Comment (victorg1991) [reply modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/renderer/LanguageSelectFragmentRenderer.java:87]: I may be crazy but I believe this changed recently :joy: Thanks for the heads up, I'm going to update it :)

**Comment by sandrodw3** [comment]
ci:test:page-management

File: modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/manager/FragmentCollectionManager.java [hunk 2]
```diff
@@ -263,8 +263,10 @@ private Map<String, Map<String, Object>> _getDynamicFragmentCollectionMaps(
 				continue;
 			}
 
-			if ((fragmentRenderer.getType() == FragmentConstants.TYPE_INPUT) &&
-				hideInputFragments) {
+			if ((fragmentRenderer.getType() == FragmentConstants.TYPE_INPUT) ||
```
Comment (ealonso) [inline modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/manager/FragmentCollectionManager.java:266]: @victorg1991 I don't like the idea of hardcoding this specific fragment in a generic class, is there any other way of doing that?

Comment (victorg1991) [reply modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/manager/FragmentCollectionManager.java:266]: I think we can check the same permission that we check here in the fragment :thinking: but we are already doing this for the submit and other fragments in the same place: https://github.com/liferay/liferay-portal/blob/178f2d7ea85e8a79d7b665ac1ffca373e1e0861d/modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/manager/FragmentCollectionManager.java#L470-L474

Comment (victorg1991) [reply modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/manager/FragmentCollectionManager.java]: Updated!

**Comment by victorg1991** [comment]
>  apps/fragment/fragment-impl/src/main/resources/META-INF/resources/tsconfig.json is outdated
wat

**Comment by victorg1991** [comment]
ci:test:sf

**Pull request closed without merging**